src/tests/conftest.py

```diff
@@ -94,10 +94,10 @@ def ip(ip_empty):
         ],
     )
     yield ip_empty
-    runsql(ip_empty, "DROP TABLE test")
-    runsql(ip_empty, "DROP TABLE author")
-    runsql(ip_empty, "DROP TABLE website")
-    runsql(ip_empty, "DROP TABLE number_table")
+    runsql(ip_empty, "DROP TABLE IF EXISTS test")
```

Comment:
why was this changed? if we're always creating a new DB, there's a guarantee that the tables exist there (maybe it's ok to add IF EXISTS but I'm wondering why)

Reply:
Without the `IF EXISTS` clause I see this error. This is coming up because now there's a logic to check if there are stored snippets similar to the one in query (typo case). This might be happening because in some testcase the  test tables are not getting created properly but then it's going back to delete it. I'm not sure exactly which testcase this happens in , also it doesn't fail on my local.

<img width="930" alt="Screenshot 2023-05-26 at 5 43 57 PM" src="https://github.com/ploomber/jupysql/assets/29096135/8a1e76e9-bb61-44fd-84f3-8d946e575c0b">